compiler/qsc_parse/src/item.rs

```diff
@@ -44,12 +44,17 @@ pub(super) fn parse(s: &mut Scanner) -> Result<Box<Item>> {
     } else if let Some(callable) = opt(s, parse_callable_decl)? {
         Box::new(ItemKind::Callable(callable))
     } else {
-        return Err(Error(ErrorKind::Rule("item", s.peek().kind, s.peek().span)));
+        let p = s.peek();
+        if !doc.is_empty() {
```

Comment:
Could you please make this an `else if`? The nested `if` breaks the established pattern here of `if ... {} else if  ... {} else {}` which made for easier reading IMO.